test/e2e/app-dir/draft-mode/app/page.tsx

```diff
@@ -1,8 +1,12 @@
 import React from 'react'
-import { draftMode } from 'next/headers'
+import { cookies, draftMode } from 'next/headers'
 
 export default function Page() {
   const { isEnabled } = draftMode()
+  let data: string | undefined
+  if (isEnabled) {
```

Comment:
I would, if I were you, seperate out the test case (`page.tsx`) into another file to verify the base case (no `cookies()`) and the special case (with `cookies()`) to ensure we have both situations covered without overlap.

Reply:
Sounds good, fixed in [1dfb2b5](https://github.com/vercel/next.js/pull/50660/commits/1dfb2b52e9de44128e893ac0d5c4a2c6b3382d32) 👍